plugins/doc_fragments/connection.py

```diff
@@ -99,4 +99,41 @@ class ModuleDocFragment(object):
           - name: ansible_hashi_vault_timeout
         type: int
         version_added: '1.3.0'
+      retries:
+        description:
+          - "Allows for retrying on errors, based on
+            the L(Retry class in the urllib3 library,https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#urllib3.util.Retry)."
+          - This collection defines recommended defaults for working with Vault.
+          - This option can be specified as a boolean, number, or dictionary.
+          - If this option is not specified or has a value of C(False) or C(0), then retries are disabled.
```

Comment:
```suggestion
          - If this option is not specified or has a value of C(false) or C(0), then retries are disabled.
```
Ansible documentation should use lower-case strings for YAML booleans.

Reply:
@felixfontein where is that defined? In YAML (including in Ansible) I usually use title case, but would like to be aware of what's recommended.

Reply:
This is not an Ansible requirement but a docs style issue